gitgud/levels/__init__.py

```diff
@@ -1,20 +1,18 @@
-from collections import OrderedDict
-
 from gitgud.levels.intro import all_challenges as intro_challenges
 from gitgud.levels.merging import all_challenges as merging_challenges
 
 from gitgud.levels.util import Level
+from gitgud.levels.util import NamedList
 
-all_levels = OrderedDict()
-
-all_levels['intro'] = Level('intro', intro_challenges)
-all_levels['merging'] = Level('merging', merging_challenges)
-
+all_levels = NamedList([
+    ('intro', Level('intro', intro_challenges)),
+    ('merging', Level('merging', merging_challenges))
+])
```

Comment:
AllLevels should use `__init__` to grab the name from each level, so specifying the names in the tuples would be unneccessary.